packages/core/src/content/content-client.ts

```diff
@@ -100,4 +106,29 @@ export class ContentClient {
 
     return this.graphqlClient.request<T>(query, variables, options);
   }
+
+  /**
+   * Retrieves the locale information for a given locale ID.
+   *
+   * @param id - The unique identifier of the locale item.
+   * @returns A promise that resolves to the locale information associated with the specified locale ID.
+   */
+  async getLocale(id: string) {
+    debug.content('Getting locale for id: %s', id);
+
+    const response = await this.get<LocaleQueryResponse>(GET_LOCALE_QUERY, { id });
+    return response.locale;
```

Comment:
what if the id is invalid or the response is empty?
should we handle those cases?